data-raw/build-read-output-refs.R

```diff
@@ -0,0 +1,78 @@
+library(dplyr)
+devtools::load_all() # can't call library(rbabylon) from within rbabylon when using pkgr
+
+###################
+# FUNCTION DEF
+###################
+
+#' Helper to re-build references for test-read-output.R
+#'
+#' Saves to `inst/test-refs/read-output-refs/`.
+#' @param .mod bbi_nonmem_model object
+build_read_output_refs <- function(.mod) {
+  .root <- file.path(get_output_dir(.mod), get_model_id(.mod))
+  out_dir <- system.file("test-refs/read-output-refs", package = "rbabylon")
```

Comment:
Similar comment to `file.path()` above. In particular, making the call as

```r
system.file("test-refs", "read-output-refs", package = "rbabylon")
```

will preserve platform independence.